packages/components/molecules/f-searchbox/CHANGELOG.md

```diff
@@ -4,12 +4,29 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
-Latest (to be added to next release)
+v4.0.0-beta.34
```

Comment:
Is this remaining in beta?

Reply:
I believe this package is still under development so most likely does still want to be in beta.

Reply:
Yes, this one is still in beta